src/render.rs

```diff
@@ -54,13 +55,16 @@ const BIN_HEADER_SIZE: u64 = 8;
 const TILE_SIZE: u64 = 8;
 const SEGMENT_SIZE: u64 = 24;
 
+// This data structure must be kept in sync with encoding::Config and the definition in
+// shaders/shared/config.wgsl.
```

Comment:
Oh, ouch. The synchronisation is in three places?

Reply:
The one in render.rs should go away. I’ll do a cleanup pass soon.

Reply:
The config struct in packed.rs will also move in the glyph run PR so I’ll fix up the comment there. I think we should land this as is.

Reply:
sgtm